### DISCUSSION TITLE
Appstore Deployment Mac / iDevice

### DISCUSSION THREAD

**Discussion by zfl0w** [bryceco/GoMap#754] [General]
Hi Bryce,

enthusiastic customer here :-) Using GoMap on Mac, iPadOS and iOS. I'm just curious, on Mac App Store I see 3.8.0 already deployed while on iDevice App Store it's still showing 3.5.5. Still works fine, but is this intended? 

Many thanks in advance! :-)

**Comment by bryceco** [comment]
It was intentional. The Mac version doesn't get its own beta releases and in this instance I went ahead and released there immediately, while the phone release gets a little more testing and refinement. 

The corresponding phone update will be available soon.
